### PR TITLE
libplacebo: 5.264.1 -> 6.338.1

### LINKED ISSUE #242503: Update request: libplacebo 5.264.1 → 6.338.1
- Package name: [libplacebo](https://code.videolan.org/videolan/libplacebo)
- Latest released version: [6.338.1](https://code.videolan.org/videolan/libplacebo/-/tags/v6.338.1)

- Current version on the unstable channel: [5.264.1](https://github.com/NixOS/nixpkgs/blob/nixos-23.05/pkgs/development/libraries/libplacebo/default.nix#L64)
- Current version on the stable/release channel: [5.264.1](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/libraries/libplacebo/default.nix#L64)

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

@tadeokondrak
@Kranzes 

-----
Note for maintainers: Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by rhododendrox**
## Description of changes

An attempt to update libplacebo since last time it was tried (see #243134) a good number of packages were broken.
A list of the changes can be found on the [tag section](https://code.videolan.org/videolan/libplacebo/-/tags) of the project repository.

This closes #242503.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by rhododendrox** [comment]
Result of `nixpkgs-review pr 269415` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages marked as broken and skipped:</summary>
  <ul>
    <li>feishin</li>
    <li>obs-studio-plugins.obs-hyperion</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
  </ul>
</details>
<details>
  <summary>148 packages failed to build:</summary>
  <ul>
    <li>MIDIVisualizer</li>
    <li>anilibria-winmaclinux</li>
    <li>arcanPackages.all-wrapped</li>
    <li>arcanPackages.arcan</li>
    <li>arcanPackages.arcan-wrapped</li>
    <li>arcanPackages.cat9-wrapped</li>
    <li>arcanPackages.durden-wrapped</li>
    <li>ffmpeg-full (arcanPackages.ffmpeg ,ffmpeg_6-full)</li>
    <li>ffmpeg-full.bin (arcanPackages.ffmpeg.bin ,ffmpeg_6-full.bin)</li>
    <li>ffmpeg-full.data (arcanPackages.ffmpeg.data ,ffmpeg_6-full.data)</li>
    <li>ffmpeg-full.dev (arcanPackages.ffmpeg.dev ,ffmpeg_6-full.dev)</li>
    <li>ffmpeg-full.doc (arcanPackages.ffmpeg.doc ,ffmpeg_6-full.doc)</li>
    <li>ffmpeg-full.lib (arcanPackages.ffmpeg.lib ,ffmpeg_6-full.lib)</li>
    <li>ffmpeg-full.man (arcanPackages.ffmpeg.man ,ffmpeg_6-full.man)</li>
    <li>arcanPackages.pipeworld-wrapped</li>
    <li>arcanPackages.prio-wrapped</li>
    <li>arcanPackages.xarcan</li>
    <li>audiobookshelf</li>
    <li>corrscope</li>
    <li>corrscope.dist</li>
    <li>deepin.deepin-music</li>
    <li>deepin.deepin-voice-note</li>
    <li>eaglemode</li>
    <li>emulationstation</li>
    <li>escrotum</li>
    <li>escrotum.dist</li>
    <li>escrotum.man</li>
    <li>ffcast</li>
    <li>ffmpeg_5-full</li>
    <li>ffmpeg_5-full.bin</li>
    <li>ffmpeg_5-full.data</li>
    <li>ffmpeg_5-full.dev</li>
    <li>ffmpeg_5-full.doc</li>
    <li>ffmpeg_5-full.lib</li>
    <li>ffmpeg_5-full.man</li>
    <li>handbrake</li>
    <li>haruna</li>
    <li>imagination</li>
    <li>kaffeine</li>
    <li>kphotoalbum</li>
    <li>libsForQt5.elisa (plasma5Packages.elisa)</li>
    <li>libsForQt5.elisa.dev (plasma5Packages.elisa.dev)</li>
    <li>libsForQt5.kdenlive (plasma5Packages.kdenlive)</li>
    <li>libsForQt5.phonon-backend-vlc (plasma5Packages.phonon-backend-vlc)</li>
    <li>soundkonverter (libsForQt5.soundkonverter)</li>
    <li>libvlc</li>
    <li>liquidsoap</li>
    <li>manim</li>
    <li>manim.dist</li>
    <li>megaglest</li>
    <li>minitube</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.advanced-scene-switcher</li>
    <li>obs-studio-plugins.droidcam-obs</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-3d-effect</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>obs-studio-plugins.obs-command-source</li>
    <li>obs-studio-plugins.obs-freeze-filter</li>
    <li>obs-studio-plugins.obs-gradient-source</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-livesplit-one</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-mute-filter</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-replay-source</li>
    <li>obs-studio-plugins.obs-rgb-levels-filter</li>
    <li>obs-studio-plugins.obs-scale-to-sound</li>
    <li>obs-studio-plugins.obs-shaderfilter</li>
    <li>obs-studio-plugins.obs-source-clone</li>
    <li>obs-studio-plugins.obs-source-record</li>
    <li>obs-studio-plugins.obs-source-switcher</li>
    <li>obs-studio-plugins.obs-teleport</li>
    <li>obs-studio-plugins.obs-text-pthread</li>
    <li>obs-studio-plugins.obs-transition-table</li>
    <li>obs-studio-plugins.obs-tuna</li>
    <li>obs-studio-plugins.obs-vaapi</li>
    <li>obs-studio-plugins.obs-vertical-canvas</li>
    <li>obs-studio-plugins.obs-vintage-filter</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.obs-websocket</li>
    <li>obs-studio-plugins.waveform</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>peek</li>
    <li>printrun</li>
    <li>printrun.dist</li>
    <li>pympress</li>
    <li>pympress.dist</li>
    <li>python310Packages.aigpy</li>
    <li>python310Packages.aigpy.dist</li>
    <li>python310Packages.auditok</li>
    <li>python310Packages.auditok.dist</li>
    <li>python310Packages.gradio</li>
    <li>python310Packages.gradio-client</li>
    <li>python310Packages.gradio-client.dist</li>
    <li>python310Packages.gradio.dist</li>
    <li>python310Packages.moderngl-window</li>
    <li>python310Packages.moderngl-window.dist</li>
    <li>python310Packages.pydub</li>
    <li>python310Packages.pydub.dist</li>
    <li>python310Packages.pyglet</li>
    <li>python310Packages.pyglet.dist</li>
    <li>python310Packages.pyrender</li>
    <li>python310Packages.pyrender.dist</li>
    <li>python310Packages.python-vlc</li>
    <li>python310Packages.python-vlc.dist</li>
    <li>python310Packages.pytmx</li>
    <li>python310Packages.pytmx.dist</li>
    <li>python310Packages.shazamio</li>
    <li>python310Packages.shazamio.dist</li>
    <li>python311Packages.aigpy</li>
    <li>python311Packages.aigpy.dist</li>
    <li>python311Packages.auditok</li>
    <li>python311Packages.auditok.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio-client</li>
    <li>python311Packages.gradio-client.dist</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.moderngl-window</li>
    <li>python311Packages.moderngl-window.dist</li>
    <li>python311Packages.pydub</li>
    <li>python311Packages.pydub.dist</li>
    <li>python311Packages.pyglet</li>
    <li>python311Packages.pyglet.dist</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyrender.dist</li>
    <li>python311Packages.python-vlc</li>
    <li>python311Packages.python-vlc.dist</li>
    <li>python311Packages.pytmx</li>
    <li>python311Packages.pytmx.dist</li>
    <li>python311Packages.shazamio</li>
    <li>python311Packages.shazamio.dist</li>
    <li>reaper</li>
    <li>restream</li>
    <li>rofi-screenshot</li>
    <li>shaq</li>
    <li>shaq.dist</li>
    <li>strawberry</li>
    <li>tidal-dl</li>
    <li>tidal-dl.dist</li>
    <li>tone</li>
    <li>video-trimmer</li>
    <li>vlc</li>
    <li>vokoscreen</li>
    <li>wtwitch</li>
  </ul>
</details>
<details>
  <summary>82 packages built:</summary>
  <ul>
    <li>adl</li>
    <li>ani-cli</li>
    <li>anime-downloader</li>
    <li>anime-downloader.dist</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>celluloid</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>deepin.dde-gsettings-schemas</li>
    <li>deepin.deepin-movie-reborn</li>
    <li>deepin.deepin-movie-reborn.dev</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ffmpeg_4-full</li>
    <li>ffmpeg_4-full.bin</li>
    <li>ffmpeg_4-full.data</li>
    <li>ffmpeg_4-full.dev</li>
    <li>ffmpeg_4-full.doc</li>
    <li>ffmpeg_4-full.lib</li>
    <li>ffmpeg_4-full.man</li>
    <li>gonic</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>hypnotix</li>
    <li>jellyfin</li>
    <li>jellyfin-ffmpeg</li>
    <li>jellyfin-ffmpeg.bin</li>
    <li>jellyfin-ffmpeg.data</li>
    <li>jellyfin-ffmpeg.dev</li>
    <li>jellyfin-ffmpeg.doc</li>
    <li>jellyfin-ffmpeg.lib</li>
    <li>jellyfin-ffmpeg.man</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-mpv-shim</li>
    <li>jellyfin-mpv-shim.dist</li>
    <li>jftui</li>
    <li>klipperscreen</li>
    <li>libplacebo</li>
    <li>libsForQt5.plasmatube (plasma5Packages.plasmatube)</li>
    <li>libsForQt5.tokodon (plasma5Packages.tokodon)</li>
    <li>linux-wallpaperengine</li>
    <li>memento</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpc-qt</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpv-unwrapped.dev</li>
    <li>mpv-unwrapped.man</li>
    <li>mpvScripts.inhibit-gnome</li>
    <li>mpvScripts.mpris</li>
    <li>mpvScripts.thumbfast</li>
    <li>mpvpaper</li>
    <li>photoqt</li>
    <li>plex-media-player</li>
    <li>plex-mpv-shim</li>
    <li>plex-mpv-shim.dist</li>
    <li>python310Packages.mpv</li>
    <li>python310Packages.mpv.dist</li>
    <li>python311Packages.mpv</li>
    <li>python311Packages.mpv.dist</li>
    <li>qimgv</li>
    <li>radioboat</li>
    <li>somafm-cli</li>
    <li>stremio</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
    <li>subtitleedit</li>
    <li>supersonic</li>
    <li>supersonic-wayland</li>
    <li>tomato-c</li>
    <li>youtube-tui</li>
    <li>ytfzf</li>
    <li>ytui-music</li>
  </ul>
</details>

**Comment by arthsmn** [comment]
> Result of `nixpkgs-review pr 269415` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
> 10 packages marked as broken and skipped:
> 148 packages failed to build:
> 82 packages built:

ffmpeg isn't building with this new version of libplacebo. See https://trac.ffmpeg.org/ticket/10644.

**Comment by rhododendrox** [comment]
While we wait for this ffmpeg issue to be resolved, we could update libplacebo only for mpv by creating a new package for the new version.
This way the other packages would still build without problems and we could update mpv to 0.37.0.

**Comment by arthsmn** [comment]
> While we wait for this ffmpeg issue to be resolved, we could update libplacebo only for mpv by creating a new package for the new version. This way the other packages would still build without problems and we could update mpv to 0.37.0.

I was thinking this too. I was going to first try ffmpeg 6.1, which might have a fix, but I'm not having success.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by AndersonTorres — APPROVED** [review]

File: pkgs/development/libraries/libplacebo/generic.nix [hunk 1]
```diff
@@ -44,7 +47,7 @@ stdenv.mkDerivation rec {
     xorg.libX11
     libunwind
     libdovi
-  ];
+  ] ++ lib.optionals (lib.versionAtLeast version "6.338") [ xxHash ];
```
[the remark this answers is not in the document]
Comment (AndersonTorres) [reply pkgs/development/libraries/libplacebo/generic.nix]: Yep

**Comment by Atemu** [comment]
Well, you're rebuilding everything that depends on ffmpeg. That'll take a day or two even on a very powerful machine. Don't bother.

Test that you can use ffmpeg with libplacebo just like before, that's good enough.

Speaking of needing to build a ton of packages, this needs to go to staging. Please consult CONTRIBUTING.md on how to do that without pinging all codeowners.

**Review by AndersonTorres — APPROVED** [review]
I will ignore some minor and polemical nitpicks for now.

File: pkgs/development/libraries/ffmpeg/generic.nix [hunk 2]
```diff
@@ -350,14 +350,14 @@ stdenv.mkDerivation (finalAttrs: {
...
-  patches = map (patch: fetchpatch patch) (extraPatches
+  patches = (map (patch: fetchpatch patch) (extraPatches
     ++ (lib.optional (lib.versionAtLeast version "6" && lib.versionOlder version "6.1")
       { # this can be removed post 6.1
         name = "fix_aacps_tablegen";
         url = "https://git.ffmpeg.org/gitweb/ffmpeg.git/patch/814178f92647be2411516bbb82f48532373d2554";
         hash = "sha256-FQV9/PiarPXCm45ldtCsxGHjlrriL8DKpn1LaKJ8owI=";
       }
-    ));
+    ))) ++ (lib.optional (lib.versionAtLeast version "5" && lib.versionOlder version "6.1") ./remove_force_icc_lut.patch); # this can be removed post 6.1
```
Comment (Atemu) [inline pkgs/development/libraries/ffmpeg/generic.nix:360]: Please use optionals.

**Review by Atemu — COMMENTED** [review]
Please change commit titles to match the contributing guidelines.

File: pkgs/development/libraries/ffmpeg/generic.nix (hunk 2 above)
Comment (rhododendrox) [reply pkgs/development/libraries/ffmpeg/generic.nix:360]: Should I change the one at line 354 as well ?
And could you elaborate on what I should change in the commit messages ?

Comment (AndersonTorres) [reply pkgs/development/libraries/ffmpeg/generic.nix:360]: This is nitpick. No one will die and the compilers are happy.

**Comment by AndersonTorres** [comment]
@ofborg build libvlc

**Review by AndersonTorres — APPROVED** [review]

File: pkgs/development/libraries/ffmpeg/generic.nix [hunk 3]
```diff
@@ -350,14 +350,14 @@ stdenv.mkDerivation (finalAttrs: {
...
-  patches = map (patch: fetchpatch patch) (extraPatches
+  patches = (map (patch: fetchpatch patch) (extraPatches
     ++ (lib.optional (lib.versionAtLeast version "6" && lib.versionOlder version "6.1")
       { # this can be removed post 6.1
         name = "fix_aacps_tablegen";
         url = "https://git.ffmpeg.org/gitweb/ffmpeg.git/patch/814178f92647be2411516bbb82f48532373d2554";
         hash = "sha256-FQV9/PiarPXCm45ldtCsxGHjlrriL8DKpn1LaKJ8owI=";
       }
-    ));
+    ))) ++ lib.optionals (lib.versionAtLeast version "5" && lib.versionOlder version "6.1") [ ./remove_force_icc_lut.patch ];
```
Comment (Atemu) [inline pkgs/development/libraries/ffmpeg/generic.nix:360]: Nit:

```suggestion
    ))) ++ lib.optionals (lib.versionAtLeast version "5" && lib.versionOlder version "6.1") [
      ./remove_force_icc_lut.patch
    ];
```

(Makes patches for future patch additions cleaner.)

**Review by Atemu — APPROVED** [review]
From the ffmpeg side, this LGTM.

Could you provide a little example of how you tested this?

**Comment by rhododendrox** [comment]
I'm using this command but I'm having no luck in trying to make this work.
```bash
ffmpeg -init_hw_device vulkan -i input.mp4 \
-vf hwupload,libplacebo=w=1440:h=1440:upscaler=ewa_lanczos:downscaler=mitchell:format=yuv420p,hwdownload,format=yuv420p \
-c:v libx264 -c:a copy output.mkv
```
Libplacebo throws an error about my gpu not having a certain feature even though it does.
It might be consequence of my gpu being from nvidia.
But I have good news too, I built ffmpeg version 6.1, tried one more time and I can confirm that it works.
So even if I can't get version 6.0(.1) to work, we can wait for ffmpeg 6.1 and then merge.

**Comment by AndersonTorres** [comment]
>  ffmpeg version 6.1

Still alpha?

**Comment by rhododendrox** [comment]
This is the pr I was talking about #271905

**Comment by Atemu** [comment]
I built mesa.drivers from the same revision of Nixpkgs and then tried this:

```
VK_DRIVER_FILES=./mesa-drivers/share/vulkan/icd.d/radeon_icd.x86_64.json ffmpeg -init_hw_device vulkan -i ~/Recordings/TMNF/test\ recording.mkv -vf hwupload,libplacebo=w=1440:h=1440:upscaler=ewa_lanczos:downscaler=mitchell:format=yuv420p,hwdownload,format=yuv420p -c:v libx264 -c:a copy -y /tmp/output.mkv
```

But I get this error:

```
[libplacebo @ 0x34c7f00] Missing device feature: hostQueryReset
[libplacebo @ 0x34c7f00] Imported Vulkan device was not created with all required features!
[libplacebo @ 0x34c7f00] Failed importing vulkan device
[libplacebo @ 0x34c7f00] Failed importing vulkan device to libplacebo!
Error while filtering: Generic error in an external library
Failed to inject frame into filter network: Generic error in an external library
Error while processing the decoded data for stream #0:0
Conversion failed!
```

My device is a 6800XT, so it certainly has that feature and vulkaninfo confirms that.

Running the same command in my regular environment with unstable's ffmpeg works.

I might be missing something in the VK loading though.

**Comment by rhododendrox** [comment]
It's exactly the same error I get (I guess it's not always nvidia's fault).
It's limited to ffmpeg version 6.0(.1) and version 6.1 works fine.

**Review by Atemu — DISMISSED** [review]
Please verify that libplacebo works in ffmpeg before and after.

**Comment by arthsmn** [comment]
#269415 was merged! Now you can have the updated ffmpeg. Just to notice, jellyfin-ffmpeg is based on version 6.0.1, so you might need to add an exception there.

**Review by arthsmn — APPROVED** [review]
LGTM.

**Review by AndersonTorres — APPROVED** [review]

**Comment by arthsmn** [comment]
What is missing for this PR? This is already preventing some updates.

**Comment by Atemu** [comment]
I need someone to demonstrate to me that libplacebo still works in ffmpeg_6 with this because I don't know how I would.

**Comment by arthsmn** [comment]
> I need someone to demonstrate to me that libplacebo still works in ffmpeg_6 with this because I don't know how I would.

You mean the 6.0 versions?

**Comment by rhododendrox** [comment]
It finally finished building and it works perfectly fine.
If anyone decides try to test this on their own be mindful you probably need to apply this commit (fc71dc9) before building, otherwise you'll get a error regarding the intel-media-sdk package and the whole build will fail.

**Comment by Atemu** [comment]
When I merged this into staging-next, what command would I have to run in order to verify it still works?

**Comment by rhododendrox** [comment]
I used this one:

```bash
ffmpeg -init_hw_device vulkan -i <input> \
-vf hwupload,libplacebo=w=1440:h=1440:upscaler=ewa_lanczos:downscaler=mitchell:format=yuv420p,hwdownload,format=yuv420p \
-c:v libx264 -c:a copy <output>
```

**Comment by rhododendrox** [comment]
Misscliked

**Review by anthr76 — APPROVED** [review]

**Comment by amaxine** [comment]
@ofborg build ffmpeg ffmpeg.passthru.tests ffmpeg-full
@ofborg build ffmpeg_5 ffmpeg_5.passthru.tests ffmpeg_5-full
@ofborg build libplacebo libplacebo.passthru.tests
@ofborg build libplacebo_5 libplacebo_5.passthru.tests
@ofborg build vlc vlc.passthru.tests

**Comment by amaxine** [comment]
I'm not a fan of merging this with all the darwin failures here, but I don't personally run nix on darwin so I cannot look into it. That seems like a blocker to merging this, at least it's why I won't hit the button :slightly_smiling_face:

**Comment by Atemu** [comment]
Note that these failures are caused by this being based on staging and ofBorg needing to basically rebuild the world in a short amount of time. Its darwin machines are not capable of that.

**Review by fpletz — APPROVED** [review]

**Comment by zeuner** [comment]
Not sure why this was merged as-is since it breaks subsequent CI due to disregarding the `pkgs/by-name` policy: https://github.com/NixOS/nixpkgs/actions/runs/7600068348/job/20697863923?pr=282473

**Comment by Atemu** [comment]
This PR's CI for by-name checks was green and the package in your error message is unrelated to this PR.

**Pull request merged**